### PR TITLE
Implement 'Practice Exercises' Screen #134

### PR DESCRIPTION
**Pull request by alouanemed**

### CODE REVIEW

File: app/src/main/java/com/kanshu/kanshu/ExerciseAdapter.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        public TextView ExFileSizeTV;
+        public ImageView ExThumbIV;
+        public ImageView DeleteBtn;
+
+        public Exercise exercise;
+
+        public UserExercisesViewHolder(View v) {
+            super(v);
+            exercise = new Exercise();
+            ExTitleTV = (TextView) v.findViewById(R.id.ex_title);
```
Comment (intrications) [inline app/src/main/java/com/kanshu/kanshu/ExerciseAdapter.java]: these variables should start with a lowercase in Java

File: app/src/main/java/com/kanshu/kanshu/ExerciseAdapter.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+public class ExerciseAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+    private List<Exercise> exercisesList;
+
+    // Provide a reference to the views for each data item
+    // Complex data items may need more than one view per item, and
+    // you provide access to all the views for a data item in a view holder
+    public class UserExercisesViewHolder extends RecyclerView.ViewHolder {
+        // each data item is just a string in this case
+
+        //Exo refers to exericice
```
Comment (intrications) [inline app/src/main/java/com/kanshu/kanshu/ExerciseAdapter.java]: I presume you mean "Ex" not "Exo".

However, I think you should just use the full word "exercise" in the variable if you have to add a comment to explain what the abbreviation means.

**Comment by intrications** [comment]
Please fix these few style issues and I can merge.

**Comment by alouanemed** [comment]
updated  :+1:

**Comment by intrications** [comment]
Thanks.

**Pull request merged**